java/bed-allocation/README.adoc

```diff
@@ -49,6 +49,28 @@ Then try _live coding_:
 
 Notice that those changes are immediately in effect.
 
+[[enterprise]]
+== Run the application with the Enterprise Edition
```

Comment:
This is not the entire description.
Even though you link to the docs, I think it's still better to mention here that:

- They need to configure username and password.
- And the way to get username and password is to [contact us](https://timefold.ai/contact) to get a license.